addons/website_slides/models/slide_channel.py

```diff
@@ -677,13 +708,14 @@ def _action_add_members(self, target_partners, **member_values):
         if to_join:
             existing_channel_partners = self.env['slide.channel.partner'].with_context(active_test=False).sudo().search([
                 ('channel_id', 'in', self.ids),
-                ('partner_id', 'in', target_partners.ids)
+                ('partner_id', 'in', target_partners.ids),
             ])
             existing_map = dict((cid, list()) for cid in self.ids)
             for item in existing_channel_partners:
                 existing_map[item.channel_id.id].append(item.partner_id.id)
 
-            archived_members = existing_channel_partners.filtered(lambda channel_partner: not channel_partner.active)
+            archived_members = existing_channel_partners.filtered(
```

Comment:
same here

Reply:
May be this is what you suggested(https://github.com/odoo/odoo/pull/79119#discussion_r1061497381)

Reply:
yes, but you changed the line, now that no other change are made on the line it's not worth it